File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/pet/PetResponseDto.java
package ps.com.viajeros.dtos.pet;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class PetResponseDto {
    private Long idPet;
    private boolean deleted;
    private boolean canil;
    private Long idSize;  // Nombre del tamaño, por ejemplo, "Chico", "Mediano", "Grande"
    private Long idType;  // Tipo del animal, por ejemplo, "Perro", "Gato"
    private Long userId;      // ID del usuario asociado
    private String name;
}

File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Mascotas/PetResponseDto.ts
export class PetResponseDto {
    idPet: number;        // ID de la mascota
    deleted: boolean;     // Indicador si la mascota está eliminada
    canil: boolean;       // Indicador si la mascota está en un canil
    idSize: string;     // Nombre del tamaño (por ejemplo: "Chico", "Mediano", "Grande")
    idType: string;     // Tipo de mascota (por ejemplo: "Perro", "Gato")
    userId: number;       // ID del usuario dueño de la mascota
    name:string
  
    constructor(
      idPet: number,
      deleted: boolean,
      name:string,
      canil: boolean,
      idSize: string,
      idType: string,
      userId: number
    ) {
      this.idPet = idPet;
      this.deleted = deleted;
      this.canil = canil;
      this.idSize = idSize;
      this.idType = idType;
      this.userId = userId;
      this.name = name;
    }
  }
  